site/src/component/Verify/Verify.tsx

```diff
@@ -32,33 +23,58 @@ const Verify: FC = () => {
     dispatch(setReviews(reviews.filter((review) => review.id !== reviewId)));
   };
 
-  if (!loaded) {
-    return <p>Loading...</p>;
-  } else if (reviews.length === 0) {
-    return <p>No reviews to display at the moment.</p>;
-  } else {
-    return (
-      <div className="content-wrapper verify-container">
-        <h1>Unverified Reviews</h1>
-        <p>Verifying a review will display the review on top of unverified reviews.</p>
-        <p>Deleting a review will remove it permanently.</p>
-        {reviews.map((review, i) => (
-          <div key={`verify-${i}`} className="verify">
-            <br />
-            <SubReview review={review}></SubReview>
-            <div className="verify-footer">
-              <Button variant="danger" className="mr-3" onClick={() => deleteReview(review.id)}>
-                Delete
+  return (
+    <ReviewItemGrid>
+      {reviews.length == 0 && <span>There are no unverified reviews</span>}
+      {reviews.map((review) => (
```

Comment:
This still feels like a lot of stuff for a `.map()`. consider making an `UnverifiedReview` component within the `AdminPage` folder and calling that.